content/en/logs/guide/logs-rbac-permissions.md

```diff
@@ -0,0 +1,389 @@
+* **`No Access` section**: all roles that does not have the `log_read_data` permission.
+
+
+## Create a restriction query
+
+Create a new restriction query defining its query filter. The new query appears in the list of restrictions with no role attached to it.
+
+{{< img src="account_management/rbac/logs_rq-create.gif" alt="Create a Restriction Query"  style="width:70%;">}}
+
+
+### Assign a role to a restriction query
+
+Pick the role wherever it stands, and assign it to the intended restriction query.
+
+*Note*: Keep in mind that a role can be assigned no more than one restriction query. Meaning, when you assign a role to a restriction query, it loses connection to the restriction query it was already attached to.
+
+{{< img src="account_management/rbac/logs_rq-assign_roles.gif" alt="Assign a role to Restriction Query"  style="width:70%;">}}
+
+Likewise, use the same "Move" interaction to grant `Unrestricted Access` to a Role, or conversely to turn it into a `No Access` role.
+
+### Check restriction queries
+
+This page won't display more than 50 restriction queries at once, and more than 50 roles per section. In case you have hundreds if not thousands of roles and restriction queries, use the filters to scope this view down:
+
+* with the restriction query filter:
+
+{{< img src="account_management/rbac/logs_rq-filter.png" alt="Filter Restriction Queries"  style="width:70%;">}}
+
+* with the role filter:
+
+{{< img src="account_management/rbac/logs_rq-view_as_role.png" alt="View as Roles"  style="width:70%;">}}
+
+* with the user filter, which is a convenient way to see what a specific user belonging to multiple roles actually has access to:
+
+{{< img src="account_management/rbac/logs_rq-view_as_user.png" alt="View as Roles"  style="width:70%;">}}
+
+[1]: https://app.datadoghq.com/logs/pipelines/data-access
+{{% /tab %}}
+{{% tab "API" %}}
+
+Revoke or grant this permission from a role via [the Roles API][1].
+Use [Restriction Queries][2] to scope the permission to a subset of Log Data.
+
+[1]: /api/#roles
+[2]: /api/?lang=bash#roles-restriction-queries-for-logs
+{{% /tab %}}
+{{< /tabs >}}
+
+## Legacy permissions
+
+These permissions are globally enabled by default for all users.
+
+[Logs Read Data](#logs-read-data) permission comes on top of these legacy permissions. For instance, say a user is restricted to the query `service:api`.
+
+* If this user has scoped [Read Index Data](#logs-read-index-data) permission on `audit` and `errors` indexes, this user only sees `service:api` logs within these indexes.
+* If this user has [livetail](#logs-livetail) permission, this users sees only sees `service:api` logs in the livetail.
+
+
+### `logs_read_index_data`
+
+Grants a role read access on some number of log indexes. Can be set either globally or limited to a subset of log indexes.
+
+To scope this permission to a subset of indexes, first remove the `logs_read_index_data` and `logs_modify_indexes` permissions on the role. Then
```

Comment:
Should there be a colon after "Then", like it's introducing the tabs? or did this sentence get chopped off?